ash/src/extensions/khr/video_queue.rs

```diff
@@ -0,0 +1,182 @@
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_video_queue.html>
+
+use crate::prelude::*;
+use crate::vk;
+use crate::RawPtr as _;
+use core::mem;
+
+impl crate::khr::video_queue::Device {
+    #[inline]
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkBindVideoSessionMemoryKHR.html>
+    pub unsafe fn bind_video_session_memory(
+        &self,
+        video_session: vk::VideoSessionKHR,
+        bind_session_memory_infos: &[vk::BindVideoSessionMemoryInfoKHR<'_>],
+    ) -> VkResult<()> {
+        (self.fp.bind_video_session_memory_khr)(
+            self.handle,
+            video_session,
+            bind_session_memory_infos.len() as u32,
+            bind_session_memory_infos.as_ptr(),
+        )
+        .result()
+    }
+
+    #[inline]
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdBeginVideoCodingKHR.html>
+    pub unsafe fn cmd_begin_video_coding(
+        &self,
+        command_buffer: vk::CommandBuffer,
+        begin_info: &vk::VideoBeginCodingInfoKHR<'_>,
+    ) {
+        (self.fp.cmd_begin_video_coding_khr)(command_buffer, begin_info);
```

Comment:
```suggestion
        (self.fp.cmd_begin_video_coding_khr)(command_buffer, begin_info)
```

Let's drop all semicolons, please. That gives us a clear error when the return types don't match, in case a return type isn't annotated with `#[must_use]` already.